### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: src/_cmake [hunk 1]
```diff
@@ -68,10 +68,10 @@ local cmake_build_options;cmake_build_options=(
     '(-Wno-error=deprecated -Werror=deprecated)-Wno-error=deprecated[Make deprecated macro and function warnings (not) errors]'
     '--warn-uninitialized[Warn about uninitialized values.]'
     '--warn-unused-vars[Warn about unused variables.]'
-    '--no-warn-unused-cli[Dont warn about command line options.]'
+    '--no-warn-unused-cli[Don't warn about command line options.]'
```
Comment (syohex) [inline src/_cmake:71]: This causes parser error. You need to use `'\''` 

```
_cmake:125: parse error near `}'
```

File: src/_drush [hunk 2]
```diff
@@ -51,7 +51,7 @@ _drush() {
     '(- *)'{-v,--verbose}'[Display extra information about the command.]' \
     '(- *)'{-y,--yes}'[Assume "yes" as answer to all prompts.]' \
     '(- *)'{-n,--no}'[Assume "no" as answer to all prompts.]' \
-    '(- *)'{-s,--simulate}'[Simulate all relevant actions (dont actually change the system).]' \
+    '(- *)'{-s,--simulate}'[Simulate all relevant actions (don't actually change the system).]' \
```
Comment (syohex) [inline src/_drush:54]: ditto

File: src/_drush [hunk 3]
```diff
@@ -114,7 +114,7 @@ case $state in
           '(--description)--description=[Filter out extensions that are provided by drupal core.]' \
           '(--destination)--destination=[The full path and filename in which the archive should be stored. If omitted, it will be saved to the drush-backups directory.]' \
           '(--no-core)--no-core[Exclude Drupal core, so the backup only contains the site specific stuff.]' \
-          '(--pipe)--pipe[Only print the destination of the archive. Useful for scripts that dont pass --destination.]' \
+          '(--pipe)--pipe[Only print the destination of the archive. Useful for scripts that don't pass --destination.]' \
```
Comment (syohex) [inline src/_drush:117]: ditto

File: src/_drush [hunk 4]
```diff
@@ -171,7 +171,7 @@ case $state in
           {-v,--verbose}'[Display extra information about the command.]' \
           {-y,--yes}'[Assume "yes" as answer to all prompts.]' \
           {-n,--no}'[Assume "no" as answer to all prompts.]' \
-          {-s,--simulate}'[Simulate all relevant actions (dont actually change the system).]' \
+          {-s,--simulate}'[Simulate all relevant actions (don't actually change the system).]' \
```
Comment (syohex) [inline src/_drush:174]: ditto

File: src/_fab [hunk 5]
```diff
@@ -75,8 +75,8 @@ _arguments -w -S -C \
   '(-p+ --password=-)'{-p+,--password=-}'[password for use with authentication and/or sudo]: :' \
   '(-H+ --hosts=-)'{-H+,--hosts=-}'[comma separated list of hosts to operate on]: :' \
   '(-R+ --roles=-)'{-R+,--roles=-}'[comma separated list of roles to operate on]: :' \
-  '(-a --no-agent)'{-a,--no-agent}'[don''t use the running SSH agent]' \
-  '(-k --no-keys)'{-k,--no-keys}'[don''t load private key files from ~/.ssh/]' \
+  '(-a --no-agent)'{-a,--no-agent}'[don't use the running SSH agent]' \
+  '(-k --no-keys)'{-k,--no-keys}'[don't load private key files from ~/.ssh/]' \
```
Comment (syohex) [inline src/_fab:79]: ditto

File: src/_git-flow [hunk 6]
```diff
@@ -137,7 +137,7 @@ __git-flow-release ()
 						-s'[Sign the release tag cryptographically]'\
 						-u'[Use the given GPG-key for the digital signature (implies -s)]'\
 						-m'[Use the given tag message]'\
-						-n'[Dont tag this release ]'\
+						-n'[Don't tag this release]'\
```
Comment (syohex) [inline src/_git-flow:140]: ditto

File: src/_pgsql_utils [hunk 7]
```diff
@@ -204,14 +204,14 @@ _psql () {
         {-R+,--record-separator=}':record separator char:' \
         {-s,--single-step}'[prompt before each query]' \
         {-S,--single-line}'[newline sends query]' \
-        {-t,--tuples-only}'[dont display header/footer]' \
+        {-t,--tuples-only}'[don't display header/footer]' \
```
Comment (syohex) [inline src/_pgsql_utils:207]: ditto

File: src/_pgsql_utils [hunk 8]
```diff
@@ -204,14 +204,14 @@ _psql () {
...
         {-T+,--table-attr=}':HTML table options:' \
         -u'[prompt for username/password]' \
         {-v+,--set=,--variable=}':set SQL variable:' \
         {-x,--expanded}'[one column per line]' \
         {-z,--field-separator-zero}'[set field separator for unaligned output to zero byte]' \
         {-0,--record-separator-zero}'[set record separator for unaligned output to zero byte]' \
-        {-X,--no-psqlrc}'[dont read ~/.psqlrc]' \
+        {-X,--no-psqlrc}'[don't read ~/.psqlrc]' \
```
Comment (syohex) [inline src/_pgsql_utils:214]: ditto

File: src/_pgsql_utils [hunk 9]
```diff
@@ -236,15 +236,15 @@ _pg_dump () {
         {-n+,--schema=}':schema to dump:_pgsql_schemas' \
         {-N+,--exclude-schema=}':schema to NOT dump:_pgsql_schemas' \
         {-o,--oids}'[dump objects identifiers for every table]' \
-        {-O,--no-owner}'[dont recreate as same owner]' \
-        {-R,--no-reconnect}'[dont output connect]' \
+        {-O,--no-owner}'[don't recreate as same owner]' \
+        {-R,--no-reconnect}'[don't output connect]' \
```
Comment (syohex) [inline src/_pgsql_utils:240]: ditto

File: src/_pgsql_utils [hunk 10]
```diff
@@ -236,15 +236,15 @@ _pg_dump () {
...
         {-s,--schema-only}'[no data, only schema]' \
         {-S+,--superuser=}':superuser name:_pgsql_users' \
         {-t+,--table=}':table to dump:_pgsql_tables' \
         {-T+,--exclude-table=}':table to NOT dump:_pgsql_tables' \
         {-v,--verbose}'[verbose mode]' \
         {-V,--version}'[display client version]' \
-        {-x,--no-{acl,privileges}}'[dont dump ACLs]' \
+        {-x,--no-{acl,privileges}}'[don't dump ACLs]' \
```
Comment (syohex) [inline src/_pgsql_utils:247]: ditto

File: src/_pgsql_utils [hunk 11]
```diff
@@ -328,12 +328,12 @@ _pg_dumpall () {
         {-g,--globals-only}'[dump only global objects, no databases]' \
         {-f+,--file=}':output file:_files' \
         {-o,--oids}'[dump objects identifiers for every table]' \
-        {-O,--no-owner}'[dont recreate as same owner]' \
+        {-O,--no-owner}'[don't recreate as same owner]' \
```
Comment (syohex) [inline src/_pgsql_utils:331]: ditto

File: src/_pgsql_utils [hunk 12]
```diff
@@ -328,12 +328,12 @@ _pg_dumpall () {
...
         {-r,--roles-only}'[no databases or tablespaces, only roles]' \
         {-s,--schema-only}'[no data, only schema]' \
         {-S+,--superuser=}':superuser name:_pgsql_users' \
         {-t,--tablespaces-only}'[no databases or roles, only tablespaces]' \
-        {-x,--no-privileges}'[dont dump ACLs]' \
+        {-x,--no-privileges}'[don't dump ACLs]' \
```
Comment (syohex) [inline src/_pgsql_utils:336]: ditto

File: src/_showoff [hunk 13]
```diff
@@ -67,20 +67,20 @@ case $state in
...
 					'(-t --style --type)'{-t,--style,--type}'=[Slide Type/Style (default: title)]:style' \
-					'(-u --no-number)'{-u,--no-number}'[Dont number the slide, use the given name verbatim]' \
+					'(-u --no-number)'{-u,--no-number}'[Don't number the slide, use the given name verbatim]' \
 					'1:title' && ret=0
 				;;
 			(create|init)
 				_arguments \
 					'(-d --slidedir)'{-d,--slidedir}'=[Sample slide directory name (default: one)]:arg' \
-					'(-n --nosamples)'{-n,--nosamples}'=[Dont create sample slides]' \
+					'(-n --nosamples)'{-n,--nosamples}'=[Don't create sample slides]' \
```
Comment (syohex) [inline src/_showoff:76]: ditto

File: src/_showoff [hunk 14]
```diff
@@ -67,20 +67,20 @@ case $state in
 					'(-n --name)'{-n,--name}'=[Slide name (name of the new slide file)]:basename' \
 					{-s,--source}'=[Include code from the given file as the slide body]:file:_files' \
 					'(-t --style --type)'{-t,--style,--type}'=[Slide Type/Style (default: title)]:style' \
-					'(-u --no-number)'{-u,--no-number}'[Dont number the slide, use the given name verbatim]' \
+					'(-u --no-number)'{-u,--no-number}'[Don't number the slide, use the given name verbatim]' \
```
Comment (syohex) [inline src/_showoff:70]: ditto

File: src/_cmake [hunk 15]
```diff
@@ -68,10 +68,10 @@ local cmake_build_options;cmake_build_options=(
...
     '-i[Run in wizard mode]'
     '-L-[List cache variables]::_values "options" "[non-advanced cache variables]" "A[advanced cache variables]" "H[non-advanced cached variables with help]" "AH[advanced cache variables with help]"'
-    '--trace[Put cmake in trace mode]'
+    '--trace[Put CMake in trace mode]'
```
Comment (syohex) [inline src/_cmake:74]: Original code use lowercase `cmake` for this option. Please don't capitalize this `cmake`

**Comment by rex4539** [comment]
Is it ok now?

**Comment by syohex** [comment]
> Is it ok now?

No

`'`(single quote) cannot be escaped in single quotes. `'Don\'t'` is invalid syntax in shell script.
It should be `'\''` as I said like `'Don'\''t'`. This consists of three characters, `'` and `\'` and `'`

- `'`  closes preceding string
- put `\'`(single quote)
- `'` starts new string

You can check whether syntax is valid by `source completion_file_path`, for example `source _cmake`.

**Comment by rex4539** [comment]
One more attempt :)

File: src/_fab [hunk 16]
```diff
@@ -63,7 +63,7 @@ output_levels=(
 
 _arguments -w -S -C \
   '(-)'{-h,--help}'[show this help message and exit]: :->noargs' \
-  '(-)'{-V,--version}'[show program''s version number and exit]: :->noargs' \
+  '(-)'{-V,--version}'[show program's version number and exit]: :->noargs' \
```
Comment (syohex) [inline src/_fab:66]: This line is not fixed yet.

**Comment by syohex** [comment]
BTW `licence` is not typo, it is British English. I suppose it should not be fixed, keep original word.

**Comment by rex4539** [comment]
I fixed `_fab` but I think the license change is correct, even just for consistency.

I mean the file is named `LICENSE` itself :)

And, for example, in `CONTRIBUTING.md` there are multiple instances where `license` was already written and I just fixed the only instance where is was written `licence`.

Let me know what you think.

**Comment by okapia** [comment]
> It should be `'\''` as I said like `'Don'\''t'`. This consists of three characters, `'` and `\'` and `'`

For what it is worth, the usual approach taken in completions distributed in zsh itself is to use double quotes for the whole string when an apostrophe is needed. A single quote inside double quotes doesn't need any additional backslashes. I wouldn't necessarily change any that are working so this is just for information.

**Comment by MaximDevoir** [comment]
I think we're okay changing `licence` to `license` so long as the changes are **outside of legal documents or statements**. So switching to `license` within documents like `CONTRIBUTING.md` should be okay.

However, if the word *licence* is used in a license file or is part of some legal text, then we shouldn't change the dialect of the language.

> I mean the file is named LICENSE itself :)

Grammarly has a good article on the difference between [Licence vs. License](https://www.grammarly.com/blog/licence-license/). Being from the US, I've always used *license* in writing. But it sounds like a filename using `license` is the correct usage.

**Comment by syohex** [comment]
> I think we're okay changing `licence` to `license` for consistency so long as the changes are **outside of legal documents or statements**. So switching to `license` within documents like `CONTRIBUTING.md` should be okay.
> 
> However, if the word _licence_ is used in a license file or is part of some legal text, then we shouldn't change the dialect of the language.

I agree

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -9,7 +9,7 @@ Contributions are welcome, just make sure you follow the guidelines:
  * Please do not just copy/paste someone else's completion, ask before.
  * Partially implemented completions are not accepted.
  * Please add a header containing authors, status and origin of the script and license header if you do not wish to use the Zsh license (example [here](src/_tox)).
- * Any reasonable open source licence is acceptable but note that we recommend the use of the Zsh license and that you should use it if you hope for the function to migrate to zsh itself.
```
Comment (MaximDevoir) [inline CONTRIBUTING.md:12]: With regards to https://github.com/zsh-users/zsh-completions/pull/701#issuecomment-611389469

This is the only instance where `licence` is okay to be `license`. All other instances in this PR should be reverted as the usage is within a legal document or text.

**Comment by rex4539** [comment]
I have reverted the rest of the `licence` instances.

File: src/_scrub [hunk 18]
```diff
@@ -28,7 +28,7 @@
 # Description
 # -----------
 #
-#  Completion script for scrub an utility which iteratively writes patterns on files or disk devices to make retrieving the data more difficult. (http://linux.die.net/man/1/scrub).
+#  Completion script for scrubbing a utility which iteratively writes patterns on files or disk devices to make retrieving the data more difficult. (http://linux.die.net/man/1/scrub).
```
Comment (syohex) [inline src/_scrub:31]: I suppose this `scrub` is command name

Comment (rex4539) [reply src/_scrub]: Good catch! Fixed.

**Pull request merged**